vcfa/sample_vcfa_test_config.json

```diff
@@ -0,0 +1,67 @@
+{
+  "//": "Sample test configuration file for VCFA",
+  "//": "Rename this file to vcfa_test_config.json and change the values to suit your system",
+  "//": "You can safely remove the comments or leave them. The parser will ignore them.",
+  "provider": {
+    "//"  : "This section contains credentials related to the vCD connection of Sys or Org user",
```

Comment:
Should "vCD connection" be "TM connection" in this particular case?

Reply:
True, changed